gui/src/app/components/entity-details/header/header.js

```diff
@@ -60,6 +68,7 @@ class EntityHeader extends React.Component {
       hash: null,
       store: null
     });
+    this.props.analytics.sendEvent(this.props.analytics.CLOSE);
```

Comment:
At this point in this file it's not clear what the category will be on this. Is this a bug? Should we maybe require that each component has a top level category defined IN the component file?

Reply:
I was trying to think about what would make sense in the data. So in this case it would result in `Store > Entity Details` and then action: `Close`. But happy to add Header into that chain if you think it's clearer.